10/src/academy/pocu/comp2500samples/w10/widgets/IDroppable.java

```diff
@@ -0,0 +1,5 @@
+package academy.pocu.comp2500samples.w10.widgets;
+
+public interface IDroppable {
+    void onDropped(Widget widget);
```

Comment:
onDrop?

(to match onDrag ... )

Drag and Drop = both active or 

Dragged and Dropped = both passive..

(same with clicked)